lib/recurly/base.php

```diff
@@ -6,6 +6,8 @@
   protected $_type;
   protected $_client;
   protected $_links;
+  protected $_values;
+  protected $_errors;
```

Comment:
Can you describe what issue this is meant to prevent?

Reply:
Converting this class to a string assumes the presence of a `$_values` array https://github.com/recurly/recurly-client-php/blob/e6a1e8e814a92d00bc7bd783953f7e4a7e5ea2c8/lib/recurly/base.php#L146

Parsing XML to object assumes the presence of an `$_errors` array https://github.com/recurly/recurly-client-php/blob/e6a1e8e814a92d00bc7bd783953f7e4a7e5ea2c8/lib/recurly/base.php#L300

To be honest, I don't think this is the very best solution. I think a possibly better solution would be to move the functions that reference `$_values` and `$_errors` into the class (Recurly_Resource) that defines their existence, but I'm concerned about breaking things. There's a bit of twisted logic and I didn't dig into it enough to determine whether there's ever a situation where something else that is not a `Recurly_Resource` would end up calling these functions that reference `$_values` and `$_errors`. So the idea was to declare them as existing, but it's up to the subclasses to define exactly what they are and assign them values.